app/views/users/show.html.erb

```diff
@@ -1,4 +1,38 @@
 <div class="container">
-  <h1><%= current_user.email.capitalize %></h1>
+  <h1>My Profile</h1>
+  <h3>User: <%= @user.email %></h3>
+
+  <div class="card-list">
+    <h5>My themes</h5>
+      <% if @user.themes %>
+        <ul>
+          <% @user.themes.each do |theme| %>
+            <li>
+              <%= link_to "#{theme.name.capitalize}", theme_path(theme) %>
+              <span class="ml-2">$ <%= theme.price %></span>
+              <%= link_to "Edit", edit_theme_path %>
```

Comment:
Should we add an argument for the redirect_to of the edit?

Reply:
No, becasuse this is under my profile page. 
1. No other user should be able to see it. 
2. 2. And the user can only see his/her listing